macos_okiomov.sh

```diff
@@ -156,8 +156,10 @@ VBoxManage setextradata "${vmname}" \
  "VBoxInternal/Devices/smc/0/Config/GetKeyFromRealSMC" 1
 VBoxManage setextradata "${vmname}" \
  "VBoxInternal2/EfiGraphicsResolution" "${resolution}"
-VBoxManage setextradata "${vmname}" \
- "VBoxInternal/Devices/efi/0/Config/DmiSystemSerial" "${serialnumber}"
+if [ -n "${serialnumber}" ]; then
+    VBoxManage setextradata "${vmname}" \
+     "VBoxInternal/Devices/efi/0/Config/DmiSystemSerial" "${serialnumber}"
```

Comment:
needs to read "**string:**${serialnumber}"

Reply:
Just curious, I have tested with both macOS and Linux (Arch) and it works without `string:` in both cases. Wonder what distro and bash version you use?

Reply:
debian sid, bash 4.4. issue #4 mentions this too

Reply:
I guess it's a VirtualBox issue, not a bash one. In my case (VirtualBox 6.0.0 r127566) it needs the string: prefix. I'm on Windows btw, trying to test it on both Cygwin and MinGW.